content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -218,6 +218,8 @@ In general, any field in a pod can be set. However, care must be taken for certa
 * Kubernetes requires the `image` field to be set before the injection has run. While you can set a specific image to override the default one,
   it is recommended to set the `image` to `auto` which will cause the sidecar injector to automatically select the image to use.
 * Some fields in `Pod` are dependent on related settings. For example, CPU request must be less than CPU limit. If both fields are not configured together, the pod may fail to start.
+* Fields `securityContext.RunAsUser` and `securityContext.RunAsGroup` might not be honored in some cases, for instance, when `TPROXY` mode is used,
+  as it requires the sidecar to run as user `0`. Override these fields with caution.
```

Comment:
We still need to handle the CNI gap and improve the UX, so for now it is basically a footgun for regular users that vendors make use of, it can have a scare warning.